src/security/BlindPeerConnector.cc

```diff
@@ -86,3 +86,13 @@ Security::BlindPeerConnector::noteNegotiationDone(ErrorState *error)
     }
 }
 
+Security::BlindPeerConnector::BlindPeerConnector(HttpRequestPointer &aRequest,
+        const Comm::ConnectionPointer &aServerConn,
+        const AsyncCallback<EncryptorAnswer> &aCallback,
+        const AccessLogEntryPointer &alp,
+        time_t timeout) :
```

Comment:
This should be constant, but I am not going to insist on restoring this in hope that this PR fiasco is not going to become a precedent for prohibiting marking parameters as constant in function implementations.

```suggestion
        const time_t timeout) :
```

Const-correctness is a good thing. We should not be arguing about those markers in function definitions or, at the very least, we should not be asking PR authors to _remove_ them.